Resources/Prototypes/Maps/ClockworkArena.yml

```
@@ -0,0 +1,14 @@
+- type: gameMap
```

yamllint

[error] wrong new line character: expected \n

(new-lines)